MaintenanceManager/MaintenanceManager.h

```diff
@@ -83,10 +96,13 @@ namespace WPEFramework {
 
                 static string g_currentMode;
```

Comment:
Please replace these static variables with members. There will be one one instance of the plugin running in thunder so we don't need to maintain state using globals or static members.

Reply:
Approving for now due to DVT2 deadlines. Please submit follow-up change to get this fixed.